hack/create-user-workspace.sh

```diff
@@ -39,5 +39,18 @@ kubectl ws create ${USER_APPSTUDIO_WORKSPACE}  --ignore-existing --type root:uni
 kubectl kustomize ${ROOT}/apibindings/${SERVICE_NAME}/ | sed "s|\${APPSTUDIO_SP_WORKSPACE}|${APPSTUDIO_SP_WORKSPACE}|g;s|\${HACBS_SP_WORKSPACE}|${HACBS_SP_WORKSPACE}|g" | \
   kubectl apply -f -
 
+echo
+echo "Patching APIBindings to accept all permission claims:"
+API_BINDINGS=$(kubectl get apibindings.apis.kcp.dev -l provided-by=infra-deployments -o name)
+for API_BINDING in ${API_BINDINGS}
+do
+  EXPORT_PERMISSION_CLAIMS=$(kubectl get ${API_BINDING} -o jsonpath='{.status.exportPermissionClaims}')
+  for EXPORT_CLAIM in $(echo ${EXPORT_PERMISSION_CLAIMS} | jq -c '.[]')
```

Comment:
must be `for EXPORT_CLAIM in $(echo "${EXPORT_PERMISSION_CLAIMS}" | jq -c '.[]')`, otherwise `echo ${EXPORT_PERMISSION_CLAIMS}` returns only 'y', not sure why

Reply:
interesting, I don't see such behavior.
But anyway, in this case using the double quote makes complete sense - fixed in: 1c3a6f5